### PR TITLE
Combine chatS2 to chatS6 with chatS

### PR DESCRIPTION
**Pull request by DrAnax**

### CODE REVIEW

File: server/src/chat_pregame.go [hunk 1]
```diff
@@ -21,32 +21,21 @@ func chatS(ctx context.Context, s *Session, d *CommandData, t *Table, cmd string
...
-
-// /s5 - Automatically start the game as soon as there are 5 players
-func chatS5(ctx context.Context, s *Session, d *CommandData, t *Table, cmd string) {
-	automaticStart(ctx, s, d, t, 5)
-}
-
-// /s6 - Automatically start the game as soon as there are 6 players
-func chatS6(ctx context.Context, s *Session, d *CommandData, t *Table, cmd string) {
-	automaticStart(ctx, s, d, t, 6)
+	switch cmd {
```
Comment (Zamiell) [inline server/src/chat_pregame.go:24]: if there are only 2 cases then shouldn't we just use an if statement?

File: server/src/chat_pregame.go [hunk 2]
```diff
@@ -21,32 +21,22 @@ func chatS(ctx context.Context, s *Session, d *CommandData, t *Table, cmd string
...
-// /s6 - Automatically start the game as soon as there are 6 players
-func chatS6(ctx context.Context, s *Session, d *CommandData, t *Table, cmd string) {
-	automaticStart(ctx, s, d, t, 6)
+	var numPlayers int
+	if cmd == "s" {
+		numPlayers = len(t.Players)+1
+	} else {
+		// Commands s2 to s6
+		if v, err := strconv.Atoi(cmd[1:]); err != nil {
+			msg := "The chat command of \"/" + cmd + "\" is not valid. Use <code>\"/help\"</code> to get a list of available commands."
```
[the remark this answers is not in the document]
Comment (DrAnax) [reply server/src/chat_pregame.go]: Technically that was DRESS not DRY (someone Else wrote that Short String)

**Comment by Zamiell** [comment]
then fix my fucked up code 😆

**Comment by DrAnax** [comment]
> then fix my fucked up code 😆

I already have

**Comment by Zamiell** [comment]
i updated it a bit to have less duplicated code

**Comment by Zamiell** [comment]
good PR, thanks

**Pull request merged**